project_event/views/project_task_view.xml

```diff
@@ -1,5 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <record id="project_event_task_tree_technical_sheet" model="ir.ui.view">
+        <field name="name">Task Tree</field>
+        <field name="model">project.task</field>
+        <field name="priority">200</field>
+        <field name="arch" type="xml">
+            <tree string="Task" default_order="date_start asc">
+                <field name="name"/>
+                    <field name="date_start"/>
+                    <field name="real_date_start"/>
+                    <field name="date_end"/>
+                    <field name="real_date_end"/>
+                    <field name="total_time"/>
+            </tree>
+        </field>
+    </record>
+    <record id="project_event_task_tree_activitys_children" model="ir.ui.view">
+        <field name="name">Task Tree</field>
+        <field name="model">project.task</field>
+        <field name="priority">150</field>
+        <field name="arch" type="xml">
+            <tree string="Task" default_order="date_start asc">
+                <field name="name"/>
```

Comment:
indentation

Reply:
Removed

Reply:
@rimbendhaou @eilst  Removed and  Changed